pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -340,8 +341,8 @@ func NewDomainFromName(name string, vmiUID types.UID) *api.Domain {
 	return domain
 }
 
-func SetupLibvirt() error {
-
+// Use named return err to allow checking Close() errors in defer
```

Comment:
We don't need this comment since `CloseIOAndCheckErr` has it.

Reply:
Fixed